README.md

```diff
@@ -1,3 +1,4 @@
 # The Best Example Ever
```

Comment:
🤔 Seems a bit _superlative_, no? Will the audience agree with this?

@Kiwimaori , any thoughts on this?

Reply:
I keep it like it is, it is just an example from the book 😉